### PR TITLE
fixes of i18n on update-an-account-orderform-configuration.md

### PR DESCRIPTION
**Pull request by GeorgeLimaDev**
It relates to [LOC-10439](https://vtex-dev.atlassian.net/browse/LOC-10439).
#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [X] Spelling and grammar accuracy (self-explanatory)

[LOC-10439]: https://vtex-dev.atlassian.net/browse/LOC-10439?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ricardoaerobr — COMMENTED** [review]
Modificar links.

**Review by karenkrieger — APPROVED** [review]
LGTM

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/update-an-account-orderform-configuration.md [hunk 1]
```diff
@@ -1,48 +1,38 @@
...
 
->⚠️ Always retrieve the current orderForm configuration before performing an update to ensure that you are modifying only the properties you want. Otherwise, old values can be overwritten. To get the current orderForm configuration for a specific account, check the [Get an account's orderForm configuration Dev. Guide](https://developers.vtex.com/docs/guides/get-an-account-orderform-configuration).
+## Updating an account orderForm configuration
 
-## Updating an account's orderForm configuration
-
-To update an account orderForm configuration, you need to use the [Update orderForm configuration](https://developers.vtex.com/vtex-rest-api/reference/updateorderformconfiguration) endpoint. In this request, you must send the `accountname` through the URL address, as shown by the example below:
+To update an account orderForm configuration, you must use the [Update orderForm configuration](https://developers.vtex.com/vtex-rest-api/reference/updateorderformconfiguration) endpoint. In this request, you must send the `accountname` in the URL address, as in the example below:
```
[the remark this answers is not in the document]
Comment (GeorgeLimaDev) [reply docs/guides/Checkout/configuration-section-api-quick-start-guides/update-an-account-orderform-configuration.md]: Done ✔

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/update-an-account-orderform-configuration.md [hunk 2]
```diff
@@ -60,16 +50,16 @@ See a request body example below:
 }
 ```
 
-After sending the request, the endpoint will return `code 204 (No Content)` and an empty response body. 
+After sending the request, the endpoint will return `code 204 (No Content)` and an empty response body.
 
-To confirm that the new orderForm settings have been applied to your account, access the [Get orderForm configuration](https://developers.vtex.com/vtex-rest-api/reference/getorderformconfiguration) endpoint again.
+To confirm that the new orderForm settings have been applied to your account, please access the [Get orderForm configuration](https://developers.vtex.com/vtex-rest-api/reference/getorderformconfiguration) endpoint again.
```
[the remark this answers is not in the document]
Comment (GeorgeLimaDev) [reply docs/guides/Checkout/configuration-section-api-quick-start-guides/update-an-account-orderform-configuration.md]: Done ✔

**Pull request merged**